packages/suite/src/components/wallet/WalletLayout/components/WalletLayoutNavigation.tsx

```diff
@@ -80,6 +80,7 @@ export const WalletLayoutNavLink = (props: WalletLayoutNavLinkProps) => {
 
 const Navigation = styled.div`
     display: flex;
+    justify-content: space-between;
```

Comment:
it breaks layout in Sign & Verify 

before:
![image](https://user-images.githubusercontent.com/3729633/193292250-d6dcac53-a876-419a-89b7-569680ded005.png)

after:
![image](https://user-images.githubusercontent.com/3729633/193292278-c6f5e83e-4ed4-4972-a4a1-a3402f524a0c.png)

Reply:
Nice catch! This was definitely not intended.

Reply:
Fixed.